src/Symfony/Component/VarDumper/Tests/Caster/ReflectionCasterTest.php

```diff
@@ -122,17 +122,31 @@ public function testReflectionParameter()
     {
         $var = new \ReflectionParameter(reflectionParameterFixture::class, 0);
 
-        $this->assertDumpMatchesFormat(
-            <<<'EOTXT'
+        if (80100 > \PHP_VERSION_ID) {
```

Comment:
This is just nitpicking to be consistent with the rest of the code. 

Please write `\PHP_VERSION_ID < 80100`

Same thing below.

Reply:
Done! But aren't we using yoda-style conditions in this repository?

Reply:
there is no variable involved here

Reply:
Understood, thanks for the explanation!